### ISSUE TITLE
Update the manage embargo via CSV bulk action to use the cocina access (rights) fields

### ISSUE THREAD

**Issue by andrewjbtw** [sul-dlss/argo#3216]
The bulk action for "Manage embargo" should be updated to use the cocina fields in the CSV. Currently, the CSV takes a list of rights that's based on the old access rights menu, before we moved to view/download/location/etc.

The CSV should be updated to take the following three headers:

- view
- download
- location
- controlledDigitalLending

As with the rights menus, "view" and "download" are required. "Location" and "controlledDigitalLending" columns will only be used when relevant.

I can provide updated help text later today listing the permitted values for each field.

**Comment by jcoyne** [comment]
@andrewjbtw I believe we are still waiting on:
> I can provide updated help text later today listing the permitted values for each field.

**Comment by andrewjbtw** [comment]
Ok, I'll file a new ticket. I just completed the Consul page listing the full set of available terms last night.
